wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitUrl.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.git.impl.nativegit;
+package org.eclipse.che.api.git;
```

Comment:
`getCodenvyTimeStampKeyLabel` method returns String label for Ssh key that will be stored on GitHub. May be It would be better to create an issue for generating different ssh key labels for Che and hosted version ?